### PR TITLE
ImportC: recognize more #define manifest constants

### PR DESCRIPTION
**Pull request by WalterBright**
The rest of the #define manifest constants.

### CODE REVIEW

File: test/compilable/testdefines.d [hunk 1]
```diff
@@ -1,4 +1,14 @@
 // EXTRA_FILES: imports/defines.c
 import imports.defines;
 
-static assert(GHI == 3);
+static assert(CC == 'c');
+static assert(I32 == 3);
```
Comment (ljmf00) [inline test/compilable/testdefines.d:5]: Can you test if typing precist, or at least do a single test for typing and on the others, don't use implicit conversion?

**Comment by ljmf00** [comment]
This is a mix of refactoring + enhancement tho.

File: test/compilable/testdefines.d (hunk 1 above)
Comment (WalterBright) [reply test/compilable/testdefines.d:5]: The asserts don't rely on implicit conversion.

**Comment by WalterBright** [comment]
It's appropriate in this case.

File: test/compilable/testdefines.d (hunk 1 above)
Comment (ljmf00) [reply test/compilable/testdefines.d:5]: > The asserts don't rely on implicit conversion.

e.g.:
```d
static assert(U32 == 4);
```
to
```d
static assert(U32 == 4U);
```

What I mean is, if we get to a point where `U32` is silently converted, some weird behaviour may happen. This is very unlikely tho, but I think we lose nothing testing those types.

**Review by dkorpel — APPROVED** [review]

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**